docs_zh_CN/faq.md

```diff
@@ -26,12 +30,48 @@
     2. 如果符号和 PyTorch 相关（例如：符号包含 caffe、aten 和 TH），请检查 PyTorch 运行时的版本是否和编译 mmcv 的一致
     3. 运行 `python mmdet/utils/collect_env.py` 以检查 PyTorch、torchvision 和 MMCV 构建和运行的环境是否相同
 
-- "RuntimeError: CUDA error: invalid configuration argument"。
+- "RuntimeError: CUDA error: invalid configuration argument"
 
     这个错误可能是由于您的 GPU 性能不佳造成的。尝试降低[THREADS_PER_BLOCK](https://github.com/open-mmlab/mmcv/blob/cac22f8cf5a904477e3b5461b1cc36856c2793da/mmcv/ops/csrc/common_cuda_helper.hpp#L10)
     的值并重新编译 mmcv。
 
-- "RuntimeError: nms is not compiled with GPU support"。
+- "RuntimeError: nms is not compiled with GPU support"
 
     这个错误是由于您的 CUDA 环境没有正确安装。
     您可以尝试重新安装您的 CUDA 环境，然后删除 mmcv/build 文件夹并重新编译 mmcv。
+
+- "Segmentation fault"
+
+    1. 检查 GCC 的版本，通常是因为 PyTorch 版本与 GCC 版本不匹配 （例如 GCC < 4.9 )，我们推荐用户使用 GCC 5.4，我们也不推荐使用 GCC 5.5， 因为有反馈 GCC 5.5 会导致 "segmentation fault" 并且切换到 GCC 5.4 就可以解决问题。
+
+    2. 检查是否正确安装了 CUDA 版本的 PyTorch 。
+
+        ```shell
+        python -c 'import torch; print(torch.cuda.is_available())'
+        ```
+
+        是否返回True。
+
+    3. 如果 `torch` 的安装是正确的，检查是否正确编译了 MMCV。
+
+        ```shell
+        python -c 'import mmcv; import mmcv.ops'
+        ```
+
+    4. 如果 MMCV 与 PyTorch 都被正确安装了，则使用 `ipdb`, `pdb` 设置断点，直接查找哪一部分的代码导致了 `segmentation fault`。
```

Comment:
1. Lack of translation for "or directly add 'print'"
2. \`segmentation fault\` and "segmentation fault", which is better?

Reply:
`segmentation fault` maybe better